src/expr/src/relation/func.rs

```diff
@@ -677,6 +683,16 @@ pub fn csv_extract(a: Datum, n_cols: usize) -> Vec<Row> {
         .collect()
 }
 
+pub fn repeat(a: Datum) -> Vec<(Row, Diff)> {
+    let n = match a {
+        Datum::Int64(n) => n as Diff,
+        Datum::Int32(n) => n as Diff,
+        Datum::Null => 0,
+        _ => panic!("Expected integer"),
+    };
```

Comment:
The rule is that you not allowed to match on `Datum` inside of an expr function (unless it is a JSON or similarly dynamically-typed function). Doing so indicates a failure of the type system. This should `unwrap_i64()` instead.

Can you also add an impl of `CastFrom<i64> for isize` to ore/cast.rs and use that here? Trying to not introduce any new `as` casts because they are dangerous.

Reply:
FWIW there are already several places where we match on `Datum` in expr functions, but sure, I can get rid of this one.

Reply:
Ugh. If you point me at them I will delete them. 🔨